test_snakes_cafe.py

```diff
@@ -1,5 +1,5 @@
 import snakes_cafe
-
+import pytest
```

Comment:
You do not need to import `pytest` because you are not using it for fixtures or exception testing.